card/lib/card/loader.rb

```diff
@@ -94,8 +96,8 @@ def load_formats
       def load_sets
         prepare_tmp_dir 'tmp/set'
         load_sets_by_pattern
-        Set.process_base_modules
-        Set.clean_empty_modules
+        Card::Set.process_base_modules
+        Card::Set.clean_empty_modules
```

Comment:
Yeah, that was done before I realized I needed the require.  Without it, these throw an error about the fact that Card::Set has an explicit prefix, but it isn't defined there.